tests/util_test.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""PyTest for :mod:`pykern.util`
+
+:copyright: Copyright (c) 2022 Bivio Software, Inc.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import pytest
+
+
+def test_is_pure_text():
+    from pykern import pkunit
+    from pykern import util
+
+    pkunit.pkeq(True, util.is_pure_text(b""))
+    pkunit.pkeq(True, util.is_pure_text(b"This is example text"))
```

Comment:
Don't test for "True" or "False". Rather, use pkok. You may consider wrapping this in a function or a loop to make the test code more readable.